tests/SelfTest/Baselines/console.sw.approved.txt

```diff
@@ -228,6 +228,9 @@ Generators.tests.cpp:<line number>: PASSED:
   REQUIRE( counter < 7 )
 with expansion:
   3 < 7
+with 1 generator
+  line:267: GENERATE(1, 2)
```

Comment:
Unfortunately, we can't print the variable name the generator value is assigned to, like in `CAPTURE`. This is why I decided to print the `GENERATE` call including the arguments. Since this can be quite long, I put the value into a separate line. Another option would be to elide long descriptions in the middle, e.g. `GENERATE(values{1, 2, ..., 10 })`.